node/utils/SyncCatalog.ts

```diff
@@ -654,33 +642,31 @@ export async function autoSyncEnd(
 
   const result: any = await masterdata.getDocument({
     dataEntity: 'lightspeed_sync',
-    fields: [SYNC_FIELDS],
+    fields: SYNC_FIELDS,
     id: params.id,
   })
 
   const jobStatus: any = await vbase.getJSON(
     'lightspeed',
     'scheduledJobComplete'
   )
-  if (jobStatus !== null && jobStatus !== undefined) {
-    if (result !== null && result !== undefined) {
-      const timeDiff = (
-        (new Date(jobStatus.completeTime).getTime() -
-          new Date(result.syncTime).getTime()) /
-        1000
-      ).toFixed(2)
-      try {
-        await masterdata.updatePartialDocument({
-          dataEntity: 'lightspeed_sync',
-          id: params.id,
-          fields: {
-            completeTime: timeDiff,
-            syncStatus: 'Completed',
-          },
-        })
-      } catch (e) {
-        console.log('Error Update Inventory =>', e)
-      }
+  if (!jobStatus || !result) {
```

Comment:
```suggestion
  if (!jobStatus || !result) {
    return
  }
  
```
This should be an early return - you don't want to run the code after this if either of these values are false. You'll also need to remove the bracket on line 672.